core/src/main/java/org/apache/iceberg/rest/RESTSessionCatalog.java

```diff
@@ -511,7 +511,6 @@ public Table loadTable(SessionContext context, TableIdentifier identifier) {
             endpoints);
 
     trackFileIO(ops);
-
```

Comment:
Can we undo this change, it's unnecessary

Reply:
Will fix this